src/components/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import { FaInstagram, FaFacebookF } from 'react-icons/fa'
+import { SiTiktok } from 'react-icons/si'
+
+export default function Footer() {
+  return (
+    <footer className="bg-brandBeige text-gray-700 text-sm py-6 px-4 mt-16">
+      <div className="max-w-5xl mx-auto flex flex-col md:flex-row justify-between items-center gap-4">
+        {/* Left: Copyright */}
+        <p className="text-center md:text-left">
+          © 2025 Daly Radiance Aesthetics. All rights reserved.
+        </p>
+
+        {/* Right: Social Icons */}
+        <div className="flex space-x-4 text-brandPink text-xl">
+          <a
+            href="https://www.instagram.com"
+            target="_blank"
```

Comment:
It's bad practice to force links to open in a new tab, it takes away choice from users and promotes clutter in the browser.